test-data/unit/deps-types.test

```diff
@@ -151,3 +151,411 @@ def h() -> None:
 <m.f> -> m, m.h
 <m.ff> -> m, m.h
 <m.g> -> m.f
+
+-- Type aliases
+
+[case testAliasDepsNormalMod]
+from mod import I
+A = I
+x: A
+[file mod.py]
+class I: pass
+[out]
+<m.A> -> m
+<m.x> -> m
+<mod.I> -> <m.x>, m
+
+[case testAliasDepsNormalModExtended]
+import a
+x: a.A
+[file a.py]
+from mod import I
+A = I
+[file mod.py]
+class I: pass
+[out]
+<m.x> -> m
+<a.A> -> m
+<a> -> m
+<mod.I> -> <m.x>, m
```

Comment:
Can you also dump dependencies in `a`? I'd like to see the dependency `<mod.I> -> a` at least.